src/main.rs

```diff
@@ -73,9 +72,9 @@ fn main() {
 
     let gene_start = SystemTime::now();
     for gene_name in sample.genes_with_mutations.clone().iter() {
-        if gene_name != target_gene {
-            continue;
-        }
+        // if gene_name != target_gene {
+        //     continue;
```

Comment:
Do all these commented sections want to be left in??

Reply:
I use them for testing as it makes it easy to pull out gene-level variation (but usually want the full genome). Commented as they're in frequent but intermittent use and I'll clean them up once I think all edge cases have been covered